python/chemiscope/input.py

```diff
@@ -581,6 +583,62 @@ def _normalize_environments(environments, structures):
     return cleaned
 
 
+def quick_settings(
+    x="",
+    y="",
+    z="",
+    color="",
+    size="",
+    symbol="",
+    trajectory=False,
+    map_settings={},
+    structure_settings={},
+):
+    """A utility function to return a ``settings`` dictionary with the most basic
+    options for a chemiscope viewer (e.g. what to show on the axes).
+
+    :param str x: The property to show on the x axis of the map.
+
+    :param str y: The property to show on the y axis of the map.
+
+    :param str z: The property to show on the z axis of the map.
+
+    :param str color: The property to use to color the map.
+
+    :param str size: The property to use to determine data point size.
+
+    :param str symbol: The (categorical) property to use to determine point markers.
+
+    :param bool trajectory: A boolean flag that sets some default options suitable
+        to view trajectory data.
+
+    :param dict map_settings: Additional settings for the map (following the
+            chemiscope settings schema).
+
+    :param dict structure_settings: Additional settings for the
+            structure viewer (following the chemiscope settings schema).
+    """
+
+    return {
+        "map": {
+            "x": {"property": x},
+            "y": {"property": y},
+            "z": {"property": z},
+            "color": {"property": color},
+            "size": {"property": size},
+            "symbol": symbol,
+        }
+        | map_settings,
```

Comment:
this syntax requires Python 3.9 (https://peps.python.org/pep-0584/), could you update the minimal required Python version in pyproject.toml & CI?

Reply:
This is top-notch PR-review! Thanks, and done.